doc/content/reference/configuration/the-things-stack.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Generic Server Options"
+description: ""
+weight: 1
+---
+
+## Global Options
+
+Under normal circumstances, only `info`, `warn` and `error` logs are printed to the console. For development, you may also want to see `debug` logs.
+
+- `log.level`: The minimum level log messages must have to be shown (default "info")
+
+## TLS Options
+
+The Things Stack serves several endpoints using TLS. TLS certificates can come from different sources.
```

Comment:
```suggestion
The Things Stack serves several endpoints using TLS. TLS configurations can come from different sources.
```

Reply:
Only certificates come from different sources, the other configuration (min TLS version, ciphers, etc.) is the same for all.